src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
-'use strict';
+"use strict";
 
-// write your code here
+const div = document.createElement("div");
+
+const success = (result) => {
+  div.className = "message";
+  div.innerText = result;
+  document.body.append(div);
+};
+
+const error = (result) => {
+  div.className = "message error-message";
+  div.innerText = result;
+  document.body.append(div);
+};
+
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector(".logo");
+
+  logo.addEventListener("click", () => {
+    resolve("Promise was resolved!");
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject("Promise was rejected!");
+  }, 3000);
+});
+
+promise1.then(success);
```

Comment:
There is no error handler added to promise1. Always add both success and error handlers to promises.